app/models/server/raw/Users.js

```diff
@@ -293,4 +293,76 @@ export class UsersRaw extends BaseRaw {
 
 		return this.update(query, update, { multi: true });
 	}
+
+	openAgentsBusinessHours(businessHourIds) {
+		const query = {
+			roles: 'livechat-agent',
+		};
+
+		const update = {
+			$set: {
+				statusLivechat: 'available',
+			},
+			$addToSet: {
+				openBusinessHours: { $each: businessHourIds },
+			},
+		};
+
+		return this.update(query, update, { multi: true });
+	}
+
+	closeAgentsBusinessHours(businessHourIds) {
```

Comment:
I think we should close the business hour for online agents only, am I right?
Let's say a big company has 3k agents, but only 10 are online, it doesn't make sense to close the business hours for offline agents, right?